packages/protocol/script/SendMessageToDelegateOwner.s.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+import "forge-std/src/Script.sol";
+import "../contracts/L2/DelegateOwner.sol";
+import "../contracts/bridge/IBridge.sol";
+import "../test/common/TestMulticall3.sol";
+
+contract SendMessageToDelegateOwner is Script {
+    address public delegateOwner = 0x904aa0aC002532f1410457484893107757683F53;
```

Comment:
Could you please verify the contract on Taiko ?

Reply:
Deployed another one and verified the code. 0x2f46c57046672e71d02ed0f0b71dfa42c7f3c488